app/src/main/res/values-en/strings.xml

```diff
@@ -1365,19 +1365,19 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="street_parking_parallel">"Parallel"</string>
     <string name="street_parking_diagonal">"Diagonal"</string>
     <string name="street_parking_perpendicular">"Perpendicular"</string>
-    <string name="street_parking_on_street">"On roadway"</string>
-    <string name="street_parking_half_on_kerb">"Partially on roadway"</string>
-    <string name="street_parking_on_kerb">"Adjacent to roadway"</string>
-    <string name="street_parking_street_side">"Adjacent to roadway only in dedicated spaces"</string>
-    <string name="street_parking_painted_area_only">"On roadway only in dedicated spaces"</string>
+    <string name="street_parking_on_street">"On-street with no curb"</string>
+    <string name="street_parking_half_on_kerb">"On-street but partially in the road"</string>
+    <string name="street_parking_on_kerb">"On-street along the curb"</string>
+    <string name="street_parking_street_side">"Off-street only in dedicated spaces"</string>
+    <string name="street_parking_painted_area_only">"On-street only in dedicated spaces"</string>
```

Comment:
Maybe this is a British/American thing, but surely you're still on the street whether it has a kerb/curb or not?
```suggestion
    <string name="street_parking_on_street">"On-street but not on the curb"</string>
    <string name="street_parking_half_on_kerb">"On-street but partially in the road"</string>
    <string name="street_parking_on_kerb">"On-street along the curb"</string>
    <string name="street_parking_street_side">"Off-street only in dedicated spaces"</string>
    <string name="street_parking_painted_area_only">"On-street only in dedicated spaces"</string>
```

Reply:
Oh, I noticed this rewording just now too. FYI, the first means that the vehicle is completely on-street, the second means that it has 2 wheels on the street and 2 wheels on the same level as the sidewalk (i.e. "beyond the curb").

Most importantly, the third means it is completely off the street but on the same level as the sidewalk, i.e. all four wheels are past the curb.

Your suggested wording sounds like especially point 3 was misunderstood(?)

Reply:
Here is a little table I compiled right now from mapillary:

<table>
<tr>
<td>street_parking_on_kerb</td>
<td><a href="https://www.mapillary.com/app/?lat=53.589913099972&lng=9.9638282&z=17&pKey=2595847270723481&focus=photo&x=0.4907449129583573&y=0.5983507653427173&zoom=0"><img src="https://user-images.githubusercontent.com/4661658/168695379-6022aea1-98ef-4700-acba-f09f422382bd.png"/></a></td>
</tr>
<tr>
<td>street_parking_half_on_kerb</td>
<td><a href="https://www.mapillary.com/app/?lat=53.583887699972&lng=9.9440541&z=17&pKey=242779424309198&x=0.5107993077680335&y=0.5395690956835644&zoom=0&focus=photo"><img src="https://user-images.githubusercontent.com/4661658/168695006-3fd5c84e-4109-4d2a-8b1b-c790cb330f82.png"/></a></td>
</tr>
<tr>
<td>street_parking_on_street</td>
<td><a href="https://www.mapillary.com/app/?lat=53.586831388889&lng=9.9672280555556&z=17&pKey=338882334530473&focus=photo&x=0.5226299240228333&y=0.5016344371372468&zoom=0"><img src="https://user-images.githubusercontent.com/4661658/168695593-0fc1e2bf-89a1-43a8-9231-d7aeca5f78c8.png"/></a></td>
</tr>
<tr>
<td>street_parking_street_side

(the difference to street_parking_on_kerb is that it is in dedicated spaces, go down the mapillary path a bit more to get what I mean)</td>
<td><a href="https://www.mapillary.com/app/?lat=53.576848099972&lng=9.9486697999722&z=19.672156152973546&pKey=291103746010338&focus=photo&x=0.5319529196160544&y=0.49578212198413985&zoom=0"><img src="https://user-images.githubusercontent.com/4661658/168695860-0959b264-181a-4ea5-99a5-1975dfd5d5c4.png"/></a></td>
</tr>
<tr>
<td>street_parking_painted_area_only</td>
<td><a href="https://www.mapillary.com/app/?lat=53.6015992&lng=9.9260812&z=17&pKey=171822434847703&focus=photo&x=0.5325532263279752&y=0.4908342469998088&zoom=0"><img src="https://user-images.githubusercontent.com/4661658/168696359-98dd7cfe-deca-4b41-86ff-2b7d36f40d73.png"/></a></td>
</tr>
</table>

Reply:
FWIW I'm not sure I'd spot the subtlety of option 3 with the current BE version either.

What about when you have lay-bys for parking in, they're not in the main flow of traffic, so I'd call them off-street, but they're at the same level of road surface (so you'd call them on-street). Is the vertical or horizontal offset more important/critical?

Reply:
> What about when you have lay-bys for parking in

Since there is maybe a confusion in wording, you better provide a picture of what you mean.

Reply:
Don’t look now, but Czechia distinguishes between parking the [<img src="https://upload.wikimedia.org/wikipedia/commons/0/01/IP11fcr.jpg" height="48" alt="" align="middle">](https://commons.wikimedia.org/wiki/File:IP11fcr.jpg) front wheels and parking the [<img src="https://upload.wikimedia.org/wikipedia/commons/6/6b/IP11gcr.jpg" height="48" alt="" align="middle">](https://commons.wikimedia.org/wiki/File:IP11gcr.jpg) right wheels on the curb.

Reply:
The tagging schema can handle that, no issue. The orientation of all pictures above is parallel to the road, the orientation can be defined separately of that.

Reply:
This is a notoriously difficult key to translate to American English, because terms like “roadway” and “curb” refer to different areas than in OSMian English. For example, here’s a cross-section of a public right-of-way from a municipal government in Utah. This is representative of American English apart from “park strip”.[^verge]

[![Ogden](https://www.ogdencity.com/ImageRepository/Document?documentId=14164)](https://www.ogdencity.com/274/The-City-Right-of-Way)

Parking off the pavement[^pavement] is generally illegal in the U.S., but there are situations where it’s not only legal but required, as seen on these standard signs:

[<img src="https://upload.wikimedia.org/wikipedia/commons/c/c9/MUTCD_R8-1.svg" width="64" alt="R8-1" align="middle">](https://commons.wikimedia.org/wiki/File:MUTCD_R8-1.svg) [<img src="https://upload.wikimedia.org/wikipedia/commons/f/ff/MUTCD-OH_R8-H1a.svg" width="64" alt="R7-H12" align="middle">](https://commons.wikimedia.org/wiki/File:MUTCD-OH_R8-H1a.svg) [<img src="https://upload.wikimedia.org/wikipedia/commons/e/ec/MUTCD-CA_R25-1.svg" width="72" alt="R25" align="middle">](https://commons.wikimedia.org/wiki/File:MUTCD-CA_R25-1.svg)

Here’s what I think would make the most sense to an American English speaker unfamiliar with OSM tagging:

* `on_street`: “In unmarked spaces on the pavement”
* `half_on_kerb`: “On the curb” or maybe “Halfway off the pavement”
* `on_kerb`: “On the grass or sidewalk” or “Off the pavement”
* `street_side`: “In marked spaces on the sidewalk”
* `painted_area_only`: “In marked spaces on the pavement”

I realize there’s some imprecision in these terms – what if the road is unpaved or the verge is dirt? However, I don’t think it’ll prevent users from choosing the right option. It would be very necessary to add explanatory comments above these strings to avoid overly literal translations into other English dialects or other languages.

[^verge]: There are a bazillion American words for `verge`. No two cities seem to have the same word for it, and none of them are mutually intelligible. Some places even call it the “parking strip” even though parking there is banned.
[^pavement]: The “pavement” meaning everything between the curbs, the opposite of what it means in British English.

Reply:
Complicating matters, the wiki doesn’t seem to [distinguish](https://wiki.openstreetmap.org/wiki/Key:parking:lane#Distinguishing_between_on_street_and_street_side) between `on_street` and `street_side` based on whether spaces are marked, as the examples above imply. Instead, [`parking=street_side`](https://wiki.openstreetmap.org/wiki/Tag:parking%3Dstreet_side) seems to be primarily about the presence of curb extensions flanking the parking spaces. I understand that the tag exists because we map the parking spaces rather than what they’re carved out of, but it’s difficult explain to a layperson. The concept of a curb extension or “bulbout” is largely unknown outside the worlds of urban planning and traffic engineering.

I most often encounter `street_side` at a park abutting the road or near a show home. Usually there’s [no curb in](https://www.openstreetmap.org/way/32616308) [the first place](https://www.openstreetmap.org/way/31879446), and [there may not even](https://www.openstreetmap.org/way/1035485550) [be marked spaces](https://www.openstreetmap.org/way/739711753).

Reply:
@1ec5 yes, `street_side` does neither require a flanking curb extension nor markings, it just requires a **dedicated** area. How it is is obvious that it is a dedicated area is not defined, could be through marking, through special paving or whatever.

Having looked through the original wording, the initial suggested wording and Minh's suggested wording, I think the original wording is fine:
- it is shortest
- it avoids "pavement", which will lead to many mistranslations because translators may not know it has opposite meanings in US- and GB-English
- [According to merriam webster](https://www.merriam-webster.com/dictionary/roadway#examples), Roadway seems to be well understood to mostly mean the part of the road that is for vehicles when looking at the examples (meaning 1b and 2). The Cambridge dictionary actually claims that the definition of "roadway" does not differ between US- and GB-English and is ["the part of the road on which vehicles drive"](https://dictionary.cambridge.org/dictionary/english/roadway) but naturally that is less of an authority

```suggestion
    <string name="street_parking_on_street">"On roadway"</string>
    <string name="street_parking_half_on_kerb">"Partially on roadway"</string>
    <string name="street_parking_on_kerb">"Adjacent to roadway"</string>
    <string name="street_parking_street_side">"Adjacent to roadway only in dedicated spaces"</string>
    <string name="street_parking_painted_area_only">"On roadway only in dedicated spaces"</string>
```

Alternatively, if you don't think so, I suggest replacing "roadway" with "street":

But according to merriam-webster, I find it can be more easily misunderstood (to mean the whole road including sidewalk):
> 1a : a thoroughfare especially in a city, town, or village that is wider than an alley or lane and that usually includes sidewalks
> b : the part of a street reserved for vehicles

```suggestion
    <string name="street_parking_on_street">"On street"</string>
    <string name="street_parking_half_on_kerb">"Partially on street"</string>
    <string name="street_parking_on_kerb">"Adjacent to street"</string>
    <string name="street_parking_street_side">"Adjacent to street only in dedicated spaces"</string>
    <string name="street_parking_painted_area_only">"On street only in dedicated spaces"</string>
```

Reply:
Also note that the app shows an illustration alongside the text. So, the text should (just) not potentially contradict the picture as it potentially with "pavement" or juggling with "curb" (not all roads have curbs, street parking values are not bound to the existence of a curb).

Reply:
Reading through this thread (thanks @westnordost for such helpful photos), it seems that the major part of the misunderstanding is on the definition of "curb".  It seems that in Germany "curb" can refer to an extended area (1-2 meters wide) next to the road.  In American usage, "curb" _only_ refers to the strip that separates the road from adjacent areas (a few centimeters wide at most).  So parking "on the curb" makes as much sense as standing on a knife's edge.  This makes the language for describing this to an American very tricky.

Reply:
The last suggestion is certainly an improvement on the starting point, so I committed that as a good step forward at least.

These are still not really right, because thinking it through both (really all) of these are on-street parking.  For example, a business would describe having "off-street parking" meaning a dedicated parking lot, vice "on-street parking" meaning along the road, which all of these are examples of.

```
    <string name="street_parking_on_street">"On street"</string>
    <string name="street_parking_half_on_kerb">"Partially on street"</string>
```

But, I can't really come up with better words to describe these concepts.

Reply:
Ah I figured it out... it's "Straddling the curb"

Reply:
I've just  changed this to "Straddling the curb" so if this works for everyone, I think we're good here!